src/dmd/errors.d

```diff
@@ -35,6 +35,41 @@ enum Classification : Color
     tip = Color.brightGreen,          /// for tip messages
 }
 
+
+enum AlwaysError
+{
+    no,
+    yes,
+}
```

Comment:
this appears to be unused

Reply:
Argh, leftover from previous experiments. Good catch.